pkgs/applications/networking/browsers/firefox/common.nix

```
@@ -322,6 +321,42 @@ buildStdenv.mkDerivation ({
     gappsWrapperArgs+=(--argv0 "$out/bin/.${binaryName}-wrapped")
   '';
 
+  # FIXME: The separateDebugInfo hook skips artifacts whose Build ID's length is not 40.
+  # But we got 16-length Build ID here. Override it to make it work.
```

I see no reason to spell "build" in "build ID" with a capital "B". The referenced PR #146275 also has it in lowercase.
```suggestion
  # FIXME: The separateDebugInfo hook skips artifacts whose build ID's length is not 40.
  # But we got 16-length build ID here. Override it to make it work.
```